spec/factories/factories.rb

```diff
@@ -47,5 +44,18 @@
     "person#{n}@example.com"
   end
 
+  factory :comment do
+    trait :for_issue do
+      association :commentable, factory: :issue
+    end
+    trait :valid do
+      title { Faker::Lorem.sentence(4) }
+      content { Faker::Lorem.sentence(10) }
+    end
 
+    trait :invalid do

```

Comment:
I'm interesting, what purpose for invalid trait? just for testing active record errors?